### DISCUSSION TITLE
How to normalise different sequencing coverage samples in CNVKit?

### DISCUSSION THREAD

**Discussion by tanbiswas** [etal/cnvkit#1024] [General]
Hi 
I'm using CNVKit to identify copy number alterations using WES data. However, our tumor samples have a coverage of 200x and corresponding normal samples have a coverage of 100x. So, is there a way to normalize the coverage before calling CNV using CNVKit?

Please let me know hoe to proceed.

**Comment by bahreinia** [comment]
Hi,
I am having the same question. We are using a panel of normal samples with 100x coverage as the reference for each tumor sample with a coverage of 300x. I would appreciate if someone could help.
Thanks

**Comment by etal** [comment]
Yes, the sequencing depth normalization is automatic.
